src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,110 @@
 import React from 'react';
 import './TodoList.scss';
+import classNames from 'classnames';
+import PropTypes from 'prop-types';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
+export class TodoList extends React.Component {
+  state = {
+    query: '',
+    status: 'all',
+  };
 
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
+  handleChange = (event, field) => {
+    this.setState({
+      [field]: event.target.value,
+    });
+  };
```

Comment:
You can use `target.value` and `target.name` for example. But I think your version is good too